components/google_sheets/sources/common.js

```diff
@@ -97,8 +99,26 @@ module.exports = {
     getWorksheetIds() {
```

Comment:
Since getWorksheetIds and getSheetId are the same in both event sources, couldn't they be removed and just use the methods in common.js if we changed the content of getWorksheetIds to "return this.worksheetIDs;" and getSheetId to "return this.sheetID;" Also, the props sheetId and worksheetId are the same in both components, so couldn't these be moved to common.js?

Reply:
The idea behind that is because I want to use `common.js` in another event source (i.e. Google Forms) which has a different way of providing this data.

In that component (ideally, it's still being discussed) we wouldn't have those props (i.e. `sheetID` and `worksheetIDs`), but rather a `formId` prop through which we would figure out the other ones. For that reason, we can't assume that `sheetID` and `worksheetIDs` are props that will always be available.